### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by kahenya-anita** [dailydotdev/daily#744] [Feature Requests]
I would like to be able to view all my badge levels.Also when they are being generated on github , can they be generated without erasing the previous one?

**Comment by idoshamun** [comment]
Hi @kahenya-anita,

If you go to your profile page, you can see how many badges you earned. I hope this makes sense :)

**Comment by kahenya-anita** [comment]
Hi, I know about that. What I really meant was to be able to click on an
individual badge and view it.Sort of like how you view when you want to
generate it or share it.

**Reply by idoshamun** [reply]
Thanks for the explanation! We don't have it now but might do it in the future :)
